ci/defs/job_configs.py

```diff
@@ -1054,3 +1054,9 @@ class JobConfigs:
         command="cd ./tests/ci && python3 libfuzzer_test_check.py 'libFuzzer tests'",
         requires=["Build (fuzzers)"],
     )
+    vector_search_stress_job = Job.Config(
+        name="Vector Search Stress",
+        runs_on=RunnerLabels.ARM_SMALL,
+        run_in_docker="clickhouse/fuzzer",
```

Comment:
Why are we calling the fuzzer image here? Did we not want to call the performance test image?